src/test/java/org/sasanlabs/service/vulnerability/xss/reflected/XSSInImgTagAttributeInjectionVulnerabilityTest.java

```diff
@@ -0,0 +1,71 @@
+package org.sasanlabs.service.vulnerability.xss.reflected;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.apache.commons.lang3.StringUtils;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.springframework.http.ResponseEntity;
+
+class XSSInImgTagAttributeInjectionVulnerabilityTest {
```

Comment:
please add `public` and also add your/author details to the class.

Reply:
rename this as well.

Reply:
Also, I think, you can modify this test for level 5 or level 7 as the newer level is not required.